tests/conftest.py

```diff
@@ -132,31 +132,25 @@ def record_schema_v2():
     return {
         'type': 'object',
         'properties': {
-            **types.ENTRY_NUMBER_V2,
-            **types.ENTRY_KEY,
-            **types.ENTRY_TIMESTAMP,
-            **types.BLOB
+            **types.RECORD_ID,
         },
-        'required': ['entry-number', 'key', 'entry-timestamp', 'blob'],
-        'additionalProperties': False
+        'required': ['_id'],
+        'additionalProperties': True
```

Comment:
I do this as the array can contain any number of fields, let me know if there's a more tightly scoped way of defining the schema?

Reply:
There is this `patternProperties` thing we use elsewhere: https://json-schema.org/latest/json-schema-validation.html#rfc.section.6.5.5

With this you I think you can express that every key is a [name](https://spec.openregister.org/v2/datatypes/name) (I don't think we check this at the moment but we should) and every value has to be a string.